src/components/calculator.js

```diff
@@ -1,34 +1,106 @@
-import React from 'react';
+/* eslint-disable react/button-has-type */
```

Comment:
- [ ] [optional] @ehmaddd great work implementing the changes requested, it would be nice if you tried fixing linter errors instead of disabling the linter check in the entire file.

Reply:
Yes surely I will do it ASAP. Thanks for the kind suggestion.